dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -358,3 +251,119 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 }
+
+namespace winrt::Microsoft::Windows::PushNotifications::factory_implementation
+{
+
+    inline constexpr auto c_maxBackoff{ 5min };
+    inline constexpr auto c_initialBackoff{ 60s };
+    inline constexpr auto c_backoffIncrement{ 60s };
+
+    const HRESULT WNP_E_NOT_CONNECTED = static_cast<HRESULT>(0x880403E8L);
+    const HRESULT WNP_E_RECONNECTING = static_cast<HRESULT>(0x880403E9L);
+    const HRESULT WNP_E_BIND_USER_BUSY = static_cast<HRESULT>(0x880403FEL);
+
+    bool IsChannelRequestRetryable(const hresult& hr)
```

Comment:
Function's not declared inline

`inline bool IsChannelRequestRetryable...`

or

`constexpr bool IsChannelRequestRetryable...`